### ISSUE TITLE
Enhancement:  No ES?  Search Database for Hashes/File Name by Default

### ISSUE THREAD

**Issue by keithjjones** [cuckoosandbox/cuckoo#1209]
If someone doesn't use ES, it would be nice to still be able to search the database for the file name or hash of the submission so you can go to that analysis.  With the problems in Issue #1178 it would be nice to have this fallback behavior.  I think when you have a large number of submissions, the "Recent" tab does not show them all so you can search for a submission that way?  I'm only seeing the last 50 or so, so unless I know the task ID to go to, it is effectively lost if ES isn't working properly.

**Comment by KillerInstinct** [comment]
FWIW I think one of the primary benefits of ES was the real time search, which can also be done in MongoDB and buffered between jQuery to Django.

**Comment by keithjjones** [comment]
Yes, losing the ability to go to a specific task because of a bug in ES is really painful when you have hundreds of samples in your Cuckoo.

**Comment by keithjjones** [comment]
If we could put reports bigger than 16MB into GridFS, that would be awesome too.  I'm losing so many analyses because of that.
